src/pages/EditProfile.js

```diff
@@ -36,6 +15,38 @@ setDescrip = (value) => {
     isInEditMode:false
   }
 
+addInfo = (name,info,description)=>{
```

Comment:
use spaces between params, like this:

```suggestion
addInfo = (name, info, description) => {
```